app/assets/src/components/Samples.jsx

```diff
@@ -1657,7 +1660,10 @@ class BackgroundModal extends React.Component {
           <div className="button-container">
             <PrimaryButton
               text="Create Collection"
-              onClick={this.handleOpen}
+              onClick={withAnalytics(
+                this.handleOpen,
+                "Samples_collection_create_button_clicked"
```

Comment:
So with this scheme, I'm assuming to find number of collection creations attempted, an analyst would want to join all the related "collection create" events? or should there be another collection create event that is always called

Reply:
An analyst could join all collection events to get stats such as collection success rate. 

There should be a backend collections event as well, `collection_created` for example, which I expect will be more commonly queried. 

It's good to have the diversity though for answering different questions about the user experience.